spec/importer/mahonia_mapper_spec.rb

```diff
@@ -13,9 +13,11 @@
       description: [description],
       keyword: ["Ellipsometry, Dielectrics -- Optical properties, Ellipsometry; high-K dielectrics; Bandgaps"],
       identifier: ["doi:10.6083/M4QN64NB"],
+      license: ["Creative Commons"],
       department: ["Dept. of Computer Science and Electrical Engineering"],
       title: ["Dielectric functions and optical bandgaps of high-K dielectrics by far ultraviolet spectroscopic ellipsometry"],
-      creator: ["Marie Curie"] }
+      creator: ["Marie Curie"],
+      embargo_date: ["1997-02-01 00:00"] }
```

Comment:
Do we need a ticket to ensure that `embargo_date` imports correctly and that embargoes lift properly after import?

Reply:
cc: @mark-dce